NuRadioReco/modules/efieldToVoltageConverter.py

```diff
@@ -74,20 +74,17 @@ def begin(self, debug=False, uncertainty=None,
         self.__pre_pulse_time = pre_pulse_time
         self.__post_pulse_time = post_pulse_time
         self.__max_upsampling_factor = 5000
-        if uncertainty is None:
-            self.__uncertainty = {}
-        else:
-            self.__uncertainty = uncertainty
+
         # some uncertainties are systematic, fix them here
-        if('sys_dx' in self.__uncertainty):
-            self.__uncertainty['sys_dx'] = np.random.normal(0, self.__uncertainty['sys_dx'])
-        if('sys_dy' in self.__uncertainty):
-            self.__uncertainty['sys_dy'] = np.random.normal(0, self.__uncertainty['sys_dy'])
-        if('sys_dz' in self.__uncertainty):
-            self.__uncertainty['sys_dz'] = np.random.normal(0, self.__uncertainty['sys_dz'])
-        if('sys_amp' in self.__uncertainty):
-            for iCh in self.__uncertainty['sys_amp'].keys():
+        self.__uncertainty = uncertainty or {}
```

Comment:
I wonder what percentage of python users finds this more intuitive than the four-line equivalent... I'm not a convert yet.

Reply:
if that is the case: ciao.